src/views/setup/components/selectEspIdf.vue

```diff
@@ -108,6 +112,16 @@ watchEffect(() => {
     store.whiteSpaceErrorIDFContainer = "";
   }
 });
+
+watch(selectedEspIdfVersion, (newVal) => {
+  clearIDfErrorStatus();
+  store.clearIdfPathError();
+  if (newVal.filename === "manual") {
```

Comment:
Seems to me that ` validatePathOnBlur` and this watch are doing the same thing. Could we remove the code of one of these 2 ?

Reply:
I couldn't find a solution in which both are not needed. 

The reason is the `@blur` event, which doesn't trigger correctly if I would use store.validateEspIdfPath, I need the verification of `selectedEspIdfVersion.value.filename === "manual"`